toolchains/BUILD

```diff
@@ -1,8 +1,10 @@
+load("@rules_cc//cc:defs.bzl", "cc_toolchain", "cc_toolchain_suite")
```

Comment:
remove cc_toolchain_suite as it's legacy and is no longer used.

Reply:
Got it!